LOCALIZE.md

```
@@ -49,7 +49,7 @@ Follow these steps to add a new word to the webapp's database:
 
 One strategy is to add words used in a particular storybook:
 
-1. Go to https://xho.elimu.ai/content/storybook/list and open a storybook that you want to add words for.
+1. Go to http://eng.elimu.ai/content/storybook/list and open a storybook that you want to add words for.
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)